src/modules/transaction/hooks/useTransactionFee/utils.js

```diff
@@ -0,0 +1,67 @@
+import { transactions } from '@liskhq/lisk-client';
+import { joinModuleAndCommand } from '@transaction/utils/moduleCommand';
+import { DEFAULT_SIGNATURE_BYTE_SIZE } from '@transaction/configuration/transactions';
+
+export const ZERO_FEE = BigInt(0);
+
+/**
+ * This is solution uses the same logic as implemented in transactions utility.
+ * We'll replace this easily with said class methods.
+ *
+ * @param {object} transaction transaction object in JSON format
+ * @param {object} paramsSchema transaction schema as retrieved from Service
+ * @param {object} auth the auth WS response data
+ * @param {array} priorities list of objects with value (number) and title (string), and selected (boolean)
+ * @param {boolean} isTxValid defines if the form was validated successfully
+ * @param {bigint} extraFee
+ *
+ * @returns {bigint} the transaction fee in Beddows
+ */
+export const computeTransactionFee = (
```

Comment:
@reyraa as discussed lets split this into three atomic hooks, which will be easy to understand and maintain

```
useComputeMinFee
useCommandFee
usePriorityAndTieBreakerFee
```

Reply:
I've removed the priority specific calculations and renamed the utility function not computeTransactionMinFee to represent the purpose. but I kept the hook name because I'm using it as the hook that populate all other fee types.